KarhooUISDK/CommonUI/VehicleCapacityView/VehicleCapacityView.swift

```diff
@@ -219,18 +219,10 @@ final class VehicleCapacityView: UIStackView {
     
     // MARK: - Public
     public func setBaggageCapacity(_ value: Int) {
-        guard value > 0 else {
-            baggageContentView.removeFromSuperview()
-            return
-        }
         baggageCapacityLabel.text = "\(value)"
     }
     
     public func setPassengerCapacity(_ value: Int) {
-        guard value > 0 else {
-            passengerCapacityContentView.removeFromSuperview()
-            return
-        }
         passengerCapacityLabel.text = "\(value)"
     }
```

Comment:
this is not a solution. value = 0 is default value in case we don't receive value from backend and cannot be presented. Only nonzero values should be presented.